driver-core/src/test/functional/com/mongodb/OperationFunctionalSpecification.groovy

```diff
@@ -443,9 +443,11 @@ class OperationFunctionalSpecification extends Specification {
             }
         }
 
-        1 * connection.command(*_) >> { throw exception }
+        1 * connection.command(*_) >> {
+            throw exception
+        }
 
-        if (serverVersionSize == 1) {
+        if (serverVersionSize == 2) {
```

Comment:
This is because of the extra `connection.getDescription().getMaxWireVersion()` checks.